taipy/gui_core/_adapters.py

```diff
@@ -84,8 +88,8 @@ def get(self):
                             (
                                 s.get_simple_label(),
                                 [t.id for t in s.tasks.values()] if hasattr(s, "tasks") else [],
-                                "" if (reason := is_submittable(s)) else f"Sequence not submittable: {reason.reasons}",
-                                is_editable(s),
+                                _get_reason(is_submittable(s), "Sequence not submittable"),
```

Comment:
No verb in the message? Ever??
No big deal - technical message kind of.

Reply:
maybe in the reasons ;-)